h/api/storage.py

```diff
@@ -37,10 +36,13 @@ def fetch_annotation(id):
     :returns: the annotation, if found, or None.
     :rtype: dict, NoneType
     """
-    return models.Annotation.fetch(id)
+    if _postgres_enabled(request):
```

Comment:
Indirection through `_postgres_enabled` probably isn't worth it any more. Just do `if request.feature('postgres_read'):`.